tests/x11/libqt5_qtbase.pm

```diff
@@ -30,26 +31,31 @@ sub run {
 
     select_console('x11');
     ensure_installed("libqt5-qttools yast2-installation", timeout => 180);
+    # we have at moment performance issue on aarch64 OSD only, so no need to change this for tumbleweed
+    # see poo#125873 for more details
+    if (is_aarch64 && is_sle) {
+        compile_qt5();
+    }
+    elsif (is_x86_64 || is_s390x || is_ppc64le || (is_aarch64 && is_tumbleweed)) {
+        designer_qt5();
```

Comment:
How  about changing the whole `if` block to something like

```
designer_qt5() unless (is_aarch64 && is_sle);
release_notes() unless (is_aarch64 && is_sle);
compile_qt5()
```

Would make it easier to understand and has the same effect. WDYT?

Reply:
yes, I like this better :)